src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -1384,16 +1387,6 @@ private void findSig(CORINFO_MODULE_STRUCT_* module, uint sigTOK, CORINFO_CONTEX
 
             Get_CORINFO_SIG_INFO(methodSig, sig);
 
-            // TODO: Replace this with a public mechanism to mark calli with SuppressGCTransition once it becomes available.
```

Comment:
I do not think that the replacement for this is part of this change, but it only affects nativeaot compiler. It is ok - we will fix in in the nativeaot branch.